app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,9 @@
+module SignUp
+  class CompletionsController < ApplicationController
+    def show; end
+
+    def update
+      redirect_to session[:saml_request_url]
```

Comment:
Could we add an analytics call here to track how many users click to continue to the SP? I think we would need the `session[:sp][:loa3]` and `@sp_name` attributes here too.